aiohttp_s3_client/xml.py

```diff
@@ -30,3 +41,37 @@ def create_complete_upload_request(parts: List[Tuple[int, str]]) -> bytes:
         b'<?xml version="1.0" encoding="UTF-8"?>' +
         ET.tostring(root, encoding="UTF-8")
     )
+
+
+def parse_list_objects(payload: bytes) -> Tuple[
+    List[AwsObjectMeta], Optional[str]
+]:
+    root = ET.fromstring(payload)
+    result = []
+    for el in root.findall(f"{{{NS}}}Contents"):
+        etag = key = last_modified = size = storage_class = None
+        for child in el:
+            tag = child.tag[child.tag.rfind("}") + 1:]
+            text = child.text
+            if text is None:
+                continue
+            if tag == "ETag":
+                etag = text
+            elif tag == "Key":
+                key = text
+            elif tag == "LastModified":
+                assert text[-1] == "Z"
+                last_modified = datetime.fromisoformat(text[:-1]).replace(
+                    tzinfo=timezone.utc
+                )
+            elif tag == "Size":
+                size = int(text)
+            elif tag == "StorageClass":
+                storage_class = intern(text)
+        if etag and key and last_modified and size is not None and \
```

Comment:
Hint: Add parentheses and reformat this as a block

Reply:
What do you use to autoformat this code? `pylava` wants me to do:
```python
        if (etag and key and last_modified and size is not None and
           storage_class):
```
Three spaces? Looks weird to me.

Reply:
I mean something like that:

```python
if (
    etag and key and
    last_modified and
    size is not None and
    storage_class
):
```

IMHO this way makes visual split between conditions and looks more readable.

Reply:
Agreed and done.